docker/Dockerfile-x86_64

```diff
@@ -1,5 +1,5 @@
-FROM centos:5.11
-MAINTAINER The ManyLinux project
+FROM centos:6.8
```

Comment:
@vitaly-krugl has a note in the PR description about centos 6 vs 5. This is a workaround for headers that are needed for capnproto that aren't available in centos 5.